rise-storage.html

```diff
@@ -1051,8 +1051,13 @@
         this._hasAttemptedGetRequest = true;
       }
       else {
-        this._startTimer();
-        this.fire("rise-cache-error", resp);
+        // Get metadata for single file on a 502 response from Rise Cache.
+        if ((e.detail.request.xhr.response.status === 502) && this._isFile) {
```

Comment:
@donnapep I think it would be good to call metadata for 5xx status code and not only for 502. What do you think? cheers